src/AcceptanceTests/TestSuiteConstraints.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using AcceptanceTesting.Support;
+
+    public partial class TestSuiteConstraints
+    {
+        public bool SupportsDtc { get; } = false;
+        public bool SupportsCrossQueueTransactions { get; } = true; // TODO: verify that this is an AtomicSendWithReceive. If not, set to false
```

Comment:
@danielmarbach just diagnosed why this test is green, and seen that we have a bug here, we are sending via even if we should not